docs/API/plain_rules.md

```diff
@@ -177,10 +178,22 @@ fields:
 Is recommended to set checkInterval at least double of reportInterval. Howeer, note that a very demanding value of
 checkInterval could impact on performance.
 
-<a name="actions"></a>
+## Non signal actions
+
+The following virtual attributes are available to be used in non signal actions:
+-   **service**: service of rule
+-   **subservice**: subservice of rule
+-   **ruleName**: name or rule
+-   **reportInterval**: time to see entity
```

Comment:
Not sure what "time to see entity" means...

Reply:
Maybe "last time entity was checked" ?

Reply:
Previous definition of condition includes:
reportInterval: mandatory, time in seconds to see an entity as silent

a silent entity is the criterion used by perseo